### PR TITLE
Allow autonetwork plugin to be used.

### PR DESCRIPTION
**Pull request by ryankilf**
Auto network should allow developers to have several homestead boxes running at once without ever having to think about IP addresses.

This solves a particular issue for us, which means we can adopt Homestead as our Vagrant VM across the company.

### CODE REVIEW

**Comment by ryankilf** [comment]
Good shout, I have made that change.

**Comment by svpernova09** [comment]
Thanks!

**Comment by browner12** [comment]
@ryankilf  can you ELI5 what this allows us to do? looks interesting, I just don't understand what it's for.

thanks

**Comment by ryankilf** [comment]
Sure, 
It means that if you put 
`ip: "autonetwork"`

in Homestead.yaml your VM will use [Vagrant Auto Network](https://github.com/oscar-stack/vagrant-auto_network) to deal with your IP addresses and your networking and all that.

It's probably less relevant if you're only running one homestead VM, but if you're running lots, and sharing them amongst your team, it's a lifesaver. 

In a previous job we used a page of IP addresses on our wiki. That's not ideal.

**Comment by browner12** [comment]
how do you keep your `/etc/hosts` file up to date then?

**Comment by ryankilf** [comment]
Autonetwork deals with it

**Comment by ryankilf** [comment]
@DojoGeekRA - I just tested this with two sites on one box today, and yes it does.
